perses/app/setup_relative_calculation.py

```diff
@@ -942,12 +996,16 @@ def _resume_run(setup_options):
             total_steps = setup_options['n_cycles']
             run_so_far = simulation.iteration
             left_to_do = total_steps - run_so_far
+            # set context caches
+            simulation.sampler_context_cache = propagation_context_cache
```

Comment:
This should be `simulation.propagation_context_cache`

Reply:
It is called `sampler_context_cache` on the openmmtools side (we changed along the way and even if the behavior is consistent the naming should be improved). Check https://github.com/choderalab/perses/pull/938#discussion_r804111821 

I will change the `openmmtools` side to have the same consistent name `propagation_context_cache` and make a bugfix release (that would be `0.21.1`), once that's done we can merge this one and pin perses to that openmmtools release. Does that sound ok?

Reply:
Why did we use `propagation_context_cache` for the keyword argument and `sampler_context_cache` for the class attribute? That is just ... confusing.

Please do the following:
* Create a new release milestone 0.21.1 for openmmtools
* Create the issue to clean up the context cache API for multistate samplers.
* To avoid any chance for confusion, **please remove all keyword arguments dealing with the context caches** from the `openmmtools` `MultiStateSampler` methods. My initial inclination was that it was a mistake to put them in because they made the API confusing and untidy, and this is confirmed by even _us_ getting confused about what they should be named. The only way users will set these is via setting the class attributes after the object is created. 
* Tag me to review the PR
* Only once I've reviewed the PR, we can merge it and cut a new release
* Then we can come back to cleaning up the perses release

Please create a new release mils

Reply:
Milestone here:
https://github.com/choderalab/openmmtools/milestone/5

Issue here:
https://github.com/choderalab/openmmtools/issues/546

Reply:
Thanks!